### PR TITLE
Add damage multiplier for draining moves

### PR DESCRIPTION
**Pull request by Ntonio36**
For ToDo list, number 6. All moves make their user recover 50% of the damage dealt, except Oblivion Wing that heals 75% of the damage.

### CODE REVIEW

**Comment by AustinXII** [comment]
.... what is your formatting

**Comment by AustinXII** [comment]
This should also be included in your other PR

**Comment by Ntonio36** [comment]
Can you just tell me what's the problem in the formatting ? Because - well, ok, those are legitimate complains- I just don't quite know what are you pointing at.. At first, well, ok, space after the if word, space before the curly bracket, etc. etc... But now what ?

File: js/data/move_data.js [hunk 1]
```diff
@@ -104,7 +104,8 @@ var MOVES_RBY = {
         bp: 100,
         type: 'Psychic',
         category: 'Special',
-        givesHealth: true
+        givesHealth: true,
+		percentHealed : 0.5
```
Comment (AustinXII) [inline js/data/move_data.js]: You don’t tab here, look at the previous line

File: js/data/move_data.js [hunk 2]
```diff
@@ -3038,7 +3047,7 @@ var MOVES_SM = $.extend(true, {}, MOVES_XY, {
     'Outrage': { zp: 190 },
     'Overheat': { zp: 195 },
     'Paleo Wave': { zp: 160 },
-    'Parabolic Charge': { bp: 65, zp:120 },
+    'Parabolic Charge' : { bp: 65, zp:120 },
```
Comment (AustinXII) [inline js/data/move_data.js]: Also what is this

**Comment by AustinXII** [comment]
It should be obvious, please follow the formatting that every other line follows in a file

File: js/data/move_data.js (hunk 2 above)
Comment (Ntonio36) [reply js/data/move_data.js]: Alright, so after double checking you were right throughout the whole process... I just am used to type that kind of stuff this way, space before the :, including other things you were and you are pointing at as formatting issues. I apologize.

Comment (AustinXII) [reply js/data/move_data.js]: It’s fine, it happens all the time

File: js/data/move_data.js [hunk 3]
```diff
@@ -104,7 +104,8 @@ var MOVES_RBY = {
         bp: 100,
         type: 'Psychic',
         category: 'Special',
-        givesHealth: true
+        givesHealth: true,
+	percentHealed: 0.5
```
Comment (AustinXII) [inline js/data/move_data.js]: Now you’re missing a space lol

File: js/data/move_data.js [hunk 4]
```diff
@@ -2483,11 +2491,12 @@ var MOVES_XY = $.extend(true, {}, MOVES_BW, {
         isPulse: true
     },
     'Overheat': { bp: 130 },
-    'Parabolic Charge': { 
+    'Parabolic Charge' : {
```
Comment (AustinXII) [inline js/data/move_data.js]: This line still

**Comment by Ntonio36** [comment]
Done.

File: js/data/move_data.js [hunk 5]
```diff
@@ -2483,11 +2491,12 @@ var MOVES_XY = $.extend(true, {}, MOVES_BW, {
         isPulse: true
     },
     'Overheat': { bp: 130 },
-    'Parabolic Charge': { 
+    'Parabolic Charge' : { 
         bp: 50,
-        type : 'Electric',
```
Comment (AustinXII) [inline js/data/move_data.js]: Also annoying that I missed this

File: js/data/move_data.js (hunk 3 above)
Comment (Ntonio36) [reply js/data/move_data.js]: So for some reason the missing space doesn't show up until I publish the commit. Everything looks finely aligned when editing the code.

**Comment by Ntonio36** [comment]
Closing PR in order to make a new one that contains both related files to merge.

**Comment by AustinXII** [comment]
If anything just edit one of your PRs don’t open another one, please

File: js/index_controls.js [hunk 6]
```diff
@@ -40,23 +40,36 @@ function calculate() {
...
         $(resultLocations[0][i].move + " + label").text(p1.moves[i].name.replace("Hidden Power", "HP"));
-        $(resultLocations[0][i].damage).text(minDisplay + " - " + maxDisplay + notation);
+        $(resultLocations[0][i].damage).text(minDisplay + " - " + maxDisplay + notation + recoveryText);
         if (maxDamage > highestDamage) {
             highestDamage = maxDamage;
             bestResult = $(resultLocations[0][i].move);
         }
         
         result = damageResults[1][i];
+	recoveryText = '';
```
Comment (AustinXII) [inline js/index_controls.js]: Missing space

File: js/index_controls.js [hunk 7]
```diff
@@ -40,23 +40,36 @@ function calculate() {
...
         minDamage = result.damage[0] * p2.moves[i].hits;
         maxDamage = result.damage[result.damage.length-1] * p2.moves[i].hits;
         minDisplay = notation === '%' ? Math.floor(minDamage * 1000 / p1.maxHP) / 10 : Math.floor(minDamage * 48 / p1.maxHP);
         maxDisplay = notation === '%' ? Math.floor(maxDamage * 1000 / p1.maxHP) / 10 : Math.floor(maxDamage * 48 / p1.maxHP);
+	if (p2.moves[i].givesHealth) {
```
Comment (AustinXII) [inline js/index_controls.js]: Here too

**Comment by Ntonio36** [comment]
Initial problem (showing percentage heal text and numbers) : Done
Formatting, re- and re-re-formatting : done
Learning how to edit multiple files and submit a single PR for all of them at once : Done
`additionalText` changed to `recoveryText` : Done.

Ready and set to merge (finally !)

**Comment by AustinXII** [comment]
Hey @Marty-D mind checking out the code to make sure the mechanics are correct?

**Comment by Marty-D** [comment]
Draining Kiss also heals 75%. If it wasn't clear yet, this repo uses 4 spaces instead of a tab; please stop mixing them randomly.

**Comment by Ntonio36** [comment]
In fact, the thing is that a tab would still make the concerned lines show slightly dis-formatted, so I had to manually add spaces to make everything look good. I'm so sleepy now, will take another look at it later in the day.

**Comment by Ntonio36** [comment]
Update : found the culprit, and it's indeed a problem in visualizing the code : I've set indentation to be equal to 4 spaces, but previewing the code shows it in an 8-space basis. I'm still going to commit the "fixes" anyways.

**Comment by AustinXII** [comment]
... you made it worse

**Comment by Ntonio36** [comment]
Well, that is when you preview it. Try to edit it and see how does it look.

**Comment by AustinXII** [comment]
It’s the same, if it looks any different on your end that an issue with your computer but I’m not merging this how it is

**Comment by Ntonio36** [comment]
You know what ? I'm done with this thing. Can't do anything if github wrongfully auto-indents something to be one space away from the rest of the same scope code, especially if there isn't anything looking wrong in Notepad++. Have a nice day.

**Comment by Zarel** [comment]
...For the record, it's not possible for GitHub to wrongfully auto-indent your code. Most likely the problem is in your code or tooling.

Specifically, the problem is that this codebase uses space indentation, while Notepad++ uses tab indentation.

Here's a guide for how to fix it:

https://stackoverflow.com/a/7471232/128023

**Comment by AustinXII** [comment]
hmm also as far as I can tell, this doesn't do anything.

**Comment by AustinXII** [comment]
whoops nevermind lol

**Pull request closed without merging**